### DISCUSSION TITLE
Display filter in the GSA Web interface for shop packs by custom label

### DISCUSSION THREAD

**Discussion by Ian-Highlander** [gameserverapp/Platform#426] [Suggestions]
Hi Max,

The shop functionality GSA offers is amazing and we use it a huge amount on our Ark cluster.

We're getting to the point where we have so many items in the shop it takes a while to scroll down the list to find items I may need to edit or check or to place new items in the right place.

We categorise ours using custom labels (see screenshot).

![image](https://user-images.githubusercontent.com/109648228/235354711-67e4412d-683d-4cf1-a1d9-be04748a958c.png)

It would be awesome if the GSA web interface had a filter option under the shop packs page so you could choose a filter by custom label and only display those items on that page at that moment. Would help with working on sections and groups of items a huge amount.

Thanks for considering.

Ian

**Comment by mvaessen** [comment]
Hi,

On the web shop this is available too, but it's hidden under the dropdown.
![Screenshot 2023-05-01 at 11 37 08](https://user-images.githubusercontent.com/13402423/235435931-14e49b85-db64-4955-9e05-a8579ac4aff9.png)

**Reply by Ian-Highlander** [reply]
Hi Max,

Sorry I may have worded it badly and not explained what I meant properly.

I meant in the GSA web interface for setting up the shop packs to make it simpler to add new packs to existing filter groups.

![image](https://user-images.githubusercontent.com/109648228/235440739-3e4fa9c5-cebc-4500-af82-0b06add214cb.png)

A "Filter by custom label" option here would be great.

Hope that makes more sense.

Cheers

Ian

**Comment by mvaessen** [comment]
Created development item #612
